packages/caravan-psbt/tsconfig.json

```diff
@@ -2,14 +2,9 @@
   "extends": "@caravan/typescript-config/base.json",
   "compilerOptions": {
     "baseUrl": ".",
-    "types": [
-      "node",
-      "jest"
-    ],
+    "types": ["node", "vitest/globals"],
```

Comment:
Shouldn't this make the imports unnecessary?

Reply:
I've removed the imports